### DISCUSSION TITLE
Camomile <-> Pd, side-by-side comparison

### DISCUSSION THREAD

**Discussion by GModal** [pierreguillot/Camomile#251] [General]
Here's a casual comparison (Camomile vs. Pd) of the same patch. It's done on Ubuntu 20.04. Camomile on the left, Pd on the right.

The Pd version also has a wrapper, to display the graph-on-parent nature of the Camomile code. That wrapper is a single object (the Camomile code abstraction).

Camomile 1.0.8
Pd 0.51-4

Yes, there are differences. Given that Camomile is tasked with converting to three different OSs, I think most of the differences are minor.

The fonts are slightly larger on the Camomile plugin, and the underscore chars in labels are removed. The lower-case descenders are retained. There are some minor alignment differences, probably due to the font size variation.

The only thing of note is the canvas label on the lower left ("gtx_banks_2.txt"), (rendered in Helvetica) has the "g" descender cut off.

![gToy_0 1 9_compare](https://user-images.githubusercontent.com/67518669/114185834-29866000-9914-11eb-8913-c720a42ff75a.png)

**Comment by pierreguillot** [comment]
Could you share the patch? I think it could be helpful to fix the GUI issues (such as https://github.com/pierreguillot/Camomile/issues/222 or https://github.com/pierreguillot/Camomile/issues/254).

**Reply by GModal** [reply]
Yes -- sorry about the delay...life and stuff. I'm dropping a zip with the Camomile example code (the Pd patch is virtually identical).

There's a short explanation (mostly about the extra folder with the rc file and presets) in the zip...

[AUCOP-gToyX_forZip.zip](https://github.com/pierreguillot/Camomile/files/6462203/AUCOP-gToyX_forZip.zip)

**Reply by GModal** [reply]
Oops, there are errors in the .txt configuration file, for two of the param lines.
